### DISCUSSION TITLE
OpenDCIM container configuration with SAML

### DISCUSSION THREAD

**Discussion by cedgnans** [opendcim/openDCIM#1580] [Q&A]
Hi, I'm creating an opendcim container to configure a SAML authentication, after deploying the container, the username dcim and password dcimadmin as seen in the youtube video didn't work. I've tried the username dcim and password dcim and got this error ![image](https://github.com/user-attachments/assets/8711923b-df31-4732-9e8d-36e1e5390e31)

**Comment by cedgnans** [comment]
When I run UPDATE fac_Config SET value='' WHERE parameter='LDAP_Debug_Password'; in the database, I still have no access to the dashboard.

**Comment by samilliken** [comment]
You have to set the authentication mode to LDAP to use the emergency LDAP password.   You have to do that to configure the SAML portion before actually switching to SAML.

 

Scott

 

From: cedgnans ***@***.***> 
Sent: Friday, November 29, 2024 5:48 AM
To: opendcim/openDCIM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [opendcim/openDCIM] OpenDCIM container configuration with SAML (Discussion #1580)

 

Hi, I'm creating an opendcim container to configure a SAML authentication, after deploying the container, the username dcim and password dcimadmin as seen in the youtube video didn't work. I've tried the username dcim and password dcim and got this error image.png (view on web) <https://github.com/user-attachments/assets/8711923b-df31-4732-9e8d-36e1e5390e31>

**Comment by cedgnans** [comment]
Yes I created the container with the environment variable OPENDCIM_AUTH to
LDAP

Le ven. 29 nov. 2024 à 15:42, Scott Milliken ***@***.***> a
écrit :
